### PR TITLE
feat: add invite link

### PR DESCRIPTION
**Pull request by djabarovgeorge**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Added an option to copy the order link for the path for users that run the service on self-hosted docker mode

- **Why this change was needed?** (You can also link to an open issue here)
Due to this reason, the application sends emails invite, while hosting it on docker self-hosted the ability to send emails is missing.

- **Other information**:

Before
image

After
image

### CODE REVIEW

File: apps/web/src/components/invites/MembersTable.tsx [hunk 1]
```diff
@@ -83,7 +91,16 @@ export function MembersTable({
                   >
                     Remove Member
                   </DropdownItem>
-                  {canResendInvite(member) ? (
+                  <When truthy={memberInvited(member)}>
+                    <DropdownItem
+                      key="resendInviteBtn"
+                      data-test-id="resend-invite-btn"
```
Comment (venarius) [inline apps/web/src/components/invites/MembersTable.tsx:97]: These should probably be changed to "copyInviteBtn" and "copy-invite-btn" 🙂

Comment (venarius) [inline apps/web/src/components/invites/MembersTable.tsx:97]: These should probably be changed to "copyInviteBtn" and "copy-invite-btn" 🙂

File: apps/web/src/components/invites/MembersTable.tsx [hunk 2]
```diff
@@ -6,6 +6,8 @@ import styled from 'styled-components';
 import * as capitalize from 'lodash.capitalize';
 import useStyles from '../../design-system/config/text.styles';
 import { MemberRole } from './MemberRole';
+import { When } from '../utils/When';
+import { useClipboard } from '@mantine/hooks';
```
Comment (venarius) [inline apps/web/src/components/invites/MembersTable.tsx:10]: Could this also be placed above the relative imports? I have also opened a PR for the import-order eslint rule now 😅

File: apps/web/src/components/invites/MembersTable.tsx (hunk 1 above)
Comment (djabarovgeorge) [reply apps/web/src/components/invites/MembersTable.tsx:97]: Most definitely is 😁

**Comment by djabarovgeorge** [comment]
@scopsy i was not aware of the new env variable indication. it will really help on the matter. 
Let me look into it. :)

**Comment by djabarovgeorge** [comment]
@scopsy Let me know if now it is more intuitive. feel free to comment regarding the copy and UX, I was not sure what is the best way to present it to the user.

Before
https://www.loom.com/share/53e60c9f9f75445cbf8b869dc584503b

After
https://www.loom.com/share/35df70968f9d45f8a5f9e09b0bb0d1d0

**Comment by scopsy** [comment]
@djabarovgeorge not quite sure about the "Provides..." text located below the input.

How about, once a user is invited, show the this component at the top or bottom:

```
  <Notification title="Invite link created">
      The invite link was successfully created. You can send it by clicking <a href="mailto:email@email.com?subject=An%20invite%20to%20Novu&body=You%20are%20invited%20to%20join%20Novu%20by%20using%20this%20link%3A%20LINK">here</a>. Or directly copying it from here:  https://link.com/data/data
  </Notification>
```

something like this from mantine, and in the mailto tag I know you can also add some dummy text to automatically be inserted there (

**Review by scopsy — APPROVED** [review]

**Comment by scopsy** [comment]
@djabarovgeorge is this can be merged? If so let's do it :)

**Pull request merged**